sources/packages/backend/libs/sims-db/src/entities/sin-validation.model.ts

```diff
@@ -47,7 +47,7 @@ export class SINValidation extends RecordDataModel {
     type: "timestamptz",
     nullable: true,
   })
-  dateReceived?: Date;
+  dateReceived?: Date | null;
```

Comment:
The dateReceived is already marked as nullable and declared as "dateReceived?". Please revert this change.